tests/scripts/arp_responder.py

```diff
@@ -26,6 +31,7 @@ def get_mac(iff):
 
 class Interface(object):
     ETH_P_ALL = 0x03
+    ETH_P_ARP = 0x806
```

Comment:
This arp_responder.py is also used by other test scripts. Will this change the behavior of arp responder and affect other tests?

Reply:
This was merged from some  previous fixes. So it should already be verified.